labelbox/client.py

```diff
@@ -1385,6 +1392,27 @@ def get_catalog_slice(self, slice_id) -> CatalogSlice:
         res = self.execute(query_str, {'id': slice_id})
         return Entity.CatalogSlice(self, res['getSavedQuery'])
 
+    def unarchive_feature_schema_node(self, ontology_id: str,
+                                      root_feature_schema_id: str) -> bool:
+        """
+        Returns true if the root feature schema node was successfully unarchived, false otherwise
+        Args:
+            root_feature_schema_id (str): The ID of the root level feature schema
+            ontology_id (str): The ID of the ontology
+        Returns:
+            bool
+        """
+        ontology_endpoint = self.rest_endpoint + "/ontologies/" + ontology_id + '/feature-schemas/' + root_feature_schema_id + '/unarchive'
```

Comment:
please encode ontology_id and root_feature_schema_id using urllib.parse.quote

Reply:
[Updated](https://github.com/Labelbox/labelbox-python/pull/939/commits/3f98a70ed7bb65f28b22272ef9b55fd7d0281ba5)